From: Michael Hanke <mih@debian.org>
Subject: Disable scimark

There are no sources of the scimark jar shipped with condor. Debian maintainer
needs to figure out whether it is worth packaging that separately, but the
project seems to be rather dead.
--- a/src/condor_starter.V6.1/CondorJavaInfo.java
+++ b/src/condor_starter.V6.1/CondorJavaInfo.java
@@ -40,7 +40,7 @@
 import java.lang.*;
 import java.util.*;
 import java.io.*;
-import jnt.scimark2.*;
+/*import jnt.scimark2.*;*/
 
 public class CondorJavaInfo {
 
@@ -73,13 +73,13 @@
 		btime = Float.valueOf(args[1]).doubleValue()/5.0;
 		if(btime>0) {
 			double result = 0;
-	                jnt.scimark2.Random r = new jnt.scimark2.Random(Constants.RANDOM_SEED);
+	        /*        jnt.scimark2.Random r = new jnt.scimark2.Random(Constants.RANDOM_SEED);
 			result += kernel.measureFFT(Constants.FFT_SIZE,btime,r);
 			result += kernel.measureSOR(Constants.SOR_SIZE,btime,r);
 			result += kernel.measureMonteCarlo(btime,r);
 			result += kernel.measureSparseMatmult(Constants.SPARSE_SIZE_M,Constants.SPARSE_SIZE_nz,btime,r);
 			result += kernel.measureLU( Constants.LU_SIZE, btime, r);
-			result /= 5.0;
+			result /= 5.0;*/
 			System.out.print( "JavaMFlops = "+result);
 			if( newmode ) {
 				System.out.println(";");
--- a/src/condor_starter.V6.1/CMakeLists.txt
+++ b/src/condor_starter.V6.1/CMakeLists.txt
@@ -26,7 +26,7 @@
 endif()
 
 # need to add all the other install targets for libexec.
-install ( FILES scimark2lib.jar CondorJavaWrapper.class CondorJavaInfo.class DESTINATION ${C_LIB} )
+install ( FILES CondorJavaWrapper.class CondorJavaInfo.class DESTINATION ${C_LIB} )
 
 if(LINUX AND WANT_GLEXEC)
 	install (FILES condor_glexec_setup condor_glexec_run condor_glexec_cleanup condor_glexec_kill condor_glexec_update_proxy DESTINATION ${C_LIBEXEC} PERMISSIONS ${CONDOR_SCRIPT_PERMS} )
